task/bq2bq/executor/main.py

```diff
@@ -30,4 +25,15 @@
         app_config.dry_run,
         app_config.job_labels,
         app_config.xcom_path,
+        on_finished_job = job_handler.handle_finished_job,
```

Comment:
can we call this `on_job_finish` ?

Reply:
How about `on_job_finished`? This is to imply that the job did finish in the past and not occurring right now. Similar to why most of the time we encounter `Done` function or method to indicate whether activity already happened or not. An example is like function `Done` in `context` or function `Done` in BigQuery client. Yes, `Do` sometime is used but only when we want to do something that hasn't happened yet.

Reply:
i think `on_job_finish` is better, as most of the event callbacks are already using similar tense. 
for eg: `on_retry_callback` is not called `on_retried_callback` , thats why

Reply:
sure

Reply:
updated